README.md

```diff
@@ -142,40 +176,154 @@ cache @matchdefault {
             BypassLockGuard true
         }
     }
+    cache_name ChangeName
+    cache_keys {
+        (host1|host2).*\.css {
+            disable_body
+            disable_host
+            disable_method
+        }
+    }
+}
+
+route /badger-configuration {
+    cache {
+        ttl 15s
+        badger {
+            configuration {
+                Dir /tmp/badger-configuration
+                ValueDir match2
+                ValueLogFileSize 16777216
```

Comment:
The Caddyfile adapter should support [`humanize`](https://github.com/dustin/go-humanize) values, i.e. `16MiB` and such.

Also, Caddyfile config idiomatically is `snake_case`, not `PascalCase`. I'd prefer if it was changed.

Reply:
That's the provider dynamic configuration it would imply to have a parser for each provider configuration and keep it up to date. Could we delay that to the next milestone with the provider externalization?

Reply:
Sure. Just wanted to point it out while I have the opportunity to.